.github/workflows/ci.yml

```diff
@@ -98,9 +109,13 @@ jobs:
           context: .
           file: Dockerfile
           platforms: |
+            linux/386
             linux/amd64
-            linux/arm64
             linux/arm/v6
-          push: true
+            linux/arm/v7
+            linux/arm64
+            linux/ppc64le
+            linux/s390x
+          push: ${{ github.event_name != 'pull_request' }}
```

Comment:
There is a check above: `github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags/')`, so it can't match on pull request I think. If you modify it, be sure to not allow Docker push on push event, as I usually create my own branches in the same repository.